apps/backend/src/main.py

```
@@ -613,14 +617,16 @@ async def root(request: Request):
         "description": "Centralized monitoring and management system for self-hosted infrastructure",
         "endpoints": {"rest_api": "/api", "health": "/health", "documentation": "/docs"},
         "external_services": {"mcp_server": "Independent MCP server via mcp_server.py"},
-        "timestamp": datetime.now(datetime.UTC).isoformat(),
+        "timestamp": datetime.now(timezone.utc).isoformat(),
     }
 
 
 # Include API routers with /api prefix
 from apps.backend.src.api import api_router
```

Ruff (0.12.2)

Module level import not at top of file

(E402)